### PR TITLE
Reverting string subsitution for updating tags. Adding method to upda…

### PR DESCRIPTION
**Pull request by mtarng**
…te complete images with no tags.

### CODE REVIEW

File: functions/pipeline/shared/db_access_v2.py [hunk 1]
```diff
@@ -111,17 +114,22 @@ def add_new_images(self,list_of_image_infos, user_id):
...
-            except Exception as e: 
+            except Exception as e:
                 print("An errors occured getting image ids: {0}".format(e))
-                raise 
+                raise
             finally: conn.close()
         return url_to_image_id_map
 
-    def update_untagged_images(self,list_of_image_ids, user_id):
+    def update_incomplete_images(self, list_of_image_ids, user_id):
```
Comment (mtarng) [inline functions/pipeline/shared/db_access_v2.py:121]: renamed

File: functions/pipeline/shared/db_access_v2.py [hunk 2]
```diff
@@ -111,17 +114,22 @@ def add_new_images(self,list_of_image_infos, user_id):
...
         #TODO: Make sure the image ids are in a TAG_IN_PROGRESS state
         self._update_images(list_of_image_ids,ImageTagState.INCOMPLETE_TAG,user_id, self._db_provider.get_connection())
         print("Updated {0} image(s) to the state {1}".format(len(list_of_image_ids),ImageTagState.INCOMPLETE_TAG.name))
 
+    def update_completed_untagged_images(self,list_of_image_ids, user_id):
```
Comment (mtarng) [inline functions/pipeline/shared/db_access_v2.py:126]: new method for visited but no tags found

File: functions/pipeline/shared/db_access_v2.py [hunk 3]
```diff
@@ -138,13 +146,14 @@ def _update_images(self, list_of_image_ids, new_image_tag_state, user_id, conn):
                 try:
                     image_ids_as_strings = [str(i) for i in list_of_image_ids]
                     images_to_update = '{0}'.format(', '.join(image_ids_as_strings))
-                    query = "UPDATE Image_Tagging_State SET TagStateId = %s, ModifiedByUser = %s, ModifiedDtim = now() WHERE ImageId IN (%s)"
-                    cursor.execute(query,(new_image_tag_state,user_id,images_to_update))
+                    # TODO: find another way to do string subsitution that doesn't break this query
+                    query = "UPDATE Image_Tagging_State SET TagStateId = {0}, ModifiedByUser = {2}, ModifiedDtim = now() WHERE ImageId IN ({1})"
```
Comment (mtarng) [inline functions/pipeline/shared/db_access_v2.py:148]: only needed to revert this line for now

File: functions/pipeline/shared/db_access_v2.py [hunk 4]
```diff
@@ -75,7 +77,8 @@ def get_new_images(self, number_of_images, user_id):
         try:
             conn = self._db_provider.get_connection()
             try:
-                cursor = conn.cursor()       
+                cursor = conn.cursor()
+                # TODO: Should we add TagStateId = INCOMPLETE_TAG also for fetching images?
```
Comment (mtarng) [inline functions/pipeline/shared/db_access_v2.py:79]: thoughts on adding priority for INCOMPLETE_TAG? Should we rank checkout priority?

**Review by andrebriggs — APPROVED** [review]

**Pull request merged**